docs/paper/dev/getting-started/project-setup.md

```diff
@@ -0,0 +1,123 @@
+
+You will land into the `build.gradle.kts` file where you can add your dependencies.
+
+### Adding Paper as a dependency
+
+To add Paper as a dependency, you will need to add the Paper repository to your `build.gradle.kts` file as well as the dependency itself.
+
+```kotlin
+repositories {
+    mavenCentral()
+    maven {
+        url = uri("https://repo.papermc.io/repository/maven-public/")
+    }
+}
+
+dependencies {
+    compileOnly("io.papermc.paper:paper-api:1.19.2-R0.1-SNAPSHOT")
+}
+
+java {
+    toolchain.languageVersion.set(JavaLanguageVersion.of(17))
+}
+```
+
+:::info
+
+To see the versions of Paper you want to use, visit our [Downloads](https://www.papermc.io/downloads) page.
+
+:::
+
+### Setting up the `src` directory
+
+To set up the `src` directory, you will need to create a new directory called `src` and then create a new directory called `main` inside of it.
+Inside of `main`, create two new directories called `java` and `resources`.
+
+It should look like this:
+
+```
+...
+src
+└── main
+    ├── java
+    └── resources
+...
+```
+
+### Setting up the `java` directory
+
+You will place your Java source files in the `java` directory. You first need to create some packages to organize your code.
+For this example, we will create three packages called `io.papermc.testplugin` and then create a class called `ExamplePlugin` inside of it.
+
+```
+...
+src
+└── main
+    ├── java
+    │   └── io
+    │       └── papermc
+    │           └── testplugin
+    │               └── ExamplePlugin.java
+    └── resources
+...
+```
+
+### The _main_ class
+
+The main class is the entry point to your plugin and will be the only class that extends `JavaPlugin` in your plugin. 
+This is an example of what your `ExamplePlugin` class could look like:
+
+```java
+package io.papermc.testplugin;
+
+import net.kyori.adventure.text.Component;
+import org.bukkit.Bukkit;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.player.PlayerJoinEvent;
+import org.bukkit.plugin.java.JavaPlugin;
+
+public class ExamplePlugin extends JavaPlugin implements Listener {
+
+    @Override
+    public void onEnable() {
+        Bukkit.getPluginManager().registerEvents(this, this);
+    }
+
+    @EventHandler
+    public void onPlayerJoin(PlayerJoinEvent event) {
+        event.getPlayer().sendMessage(Component.text("Hello, " + event.getPlayer().getName() + "!"));
+    }
+
+}
+```
+
+### Setting up the `resources`
+
+The `resources` directory is where you will place your plugin's `plugin.yml` file. See the [Plugin YML](/paper/dev/plugin-yml) page for more information.
+
+### Conclusion
+
+You should now have a project set up with Paper as a dependency.
+All you have left to do now is to compile your plugin and run it on a paper server.
+
+:::info
+    
+If you are using IntelliJ, you can use the Gradle GUI `Build` menu to compile your plugin - found on the top right of your IDE.
```

Comment:
The output jar will be in `build/libs`, which people who are unfamiliar with gradle may not expect. This block should mention where the jar can be found.